### PR TITLE
Added Matreshka.js

### PR DESCRIPTION
**Pull request by finom**
Hi there! Matreshka is the awesome JavaScript framework, which going to be updated to 1.0 soon. The website will be ready in a week. Thank you!

### CODE REVIEW

File: files/matreshka/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "matreshka",
+  "repo": "finom/matreshka",
+  "files": {
+    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
```
Comment (jimaek) [inline files/matreshka/update.json]: I dont see the map file in the repo

Comment (finom) [inline files/matreshka/update.json]: Sorry, just pushed.

**Comment by jimaek** [comment]
Thank you

**Comment by finom** [comment]
Wow! Fast service! Thank you.

**Comment by finom** [comment]
The only question is how jsDelivr detects version if packageManager is github?

**Comment by jimaek** [comment]
It uses your tags https://github.com/finom/matreshka/releases

Thanks for using jsDelivr :)

**Comment by finom** [comment]
Does it mean I need to use semver for tags? 0.3.0 instead of v0.3.0?

**Comment by jimaek** [comment]
No need, the bot will correct it automatically

**Comment by finom** [comment]
Cool!

**Pull request merged**